README.adoc

```diff
@@ -0,0 +1,154 @@
+= Hello World addon
```

Comment:
Any specific reason for using ascidoc for the README instead of reST ? Most if not all our README & docs files are reStructuredText and it has been working fine so far as far as I can tell.

Reply:
I simply used what I am familiar with.

Reply:
Changed.